programs/Makefile

```diff
@@ -262,9 +262,7 @@ mandir      ?= $(datarootdir)/man
 man1dir     ?= $(mandir)/man1
 
 ifneq (,$(filter $(shell uname),OpenBSD FreeBSD NetBSD DragonFly SunOS))
-MANDIR  ?= $(PREFIX)/man/man1
-else
-MANDIR  ?= $(man1dir)
+MANDIR  ?= $(PREFIX)/man
```

Comment:
This construction prevents using `mandir` nor `man1dir` as an input of this Makefile.
These variables are recommended by the [GNU Makefile coding standards](https://www.gnu.org/prep/standards/html_node/Directory-Variables.html#Directory-Variables).

It also changes the destination directory for non-BSD targets, 
from `$(prefix)/share/man/man1` to `$(prefix)/man/man1`.

Reply:
Yeah, this was unintentional.